_config.yml

```diff
@@ -8,6 +8,8 @@ languages:
   - en
   - es
   - fr
+# Do not show Chinese in dropdown yet
+# - zh
```

Comment:
I wonder if we should configure it by default, and then exclude via config override in Cloud.gov Pages admin for production site? That way we can at least see it in the preview.

Reply:
Yep, changed to do that.